examples/set-annotations/imperative/.expected/diff.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/resources.yaml b/resources.yaml
```

Comment:
General note:

We should have better file names than `resources`. Similar to naming this `configs`, it convey no information as everything is a resource, so it should be avoided.

Reply:
I'm not sure if `krm-resources.yaml` would be a better name.
Any suggested names?